src/index.html

```diff
@@ -17,11 +20,306 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`styles/style.scss`). Ensure that the SCSS is compiled to CSS and the link is updated to point to the compiled CSS file, as browsers do not natively understand SCSS.